### PR TITLE
Add limits value to message

### LINKED ISSUE #860: Include Violated Threshold Value in Limits Messages
Per @jmthomas:
Another user asked if limits notifications could show the limits setting as well as the violation. Currently we have:

INST HEALTH_STATUS TEMP1 = -82.04769999999999 is RED_LOW (2023/10/03 22:36:29.091)

So maybe something like:

INST HEALTH_STATUS TEMP1 = -82.04769999999999 is RED_LOW (-80.0) at 2023/10/03 22:36:29.091

### PR DESCRIPTION
**Pull request by jmthomas**
closes #860

### CODE REVIEW

**Comment by jmthomas** [comment]
Screenshot 2024-10-19 at 2 18 34 PM

File: openc3/lib/openc3/microservices/decom_microservice.rb [hunk 1]
```diff
@@ -137,10 +138,23 @@ def limits_change_callback(packet, item, old_limits_state, value, log_change)
...
+            message += " (#{values[1]} to #{values[2]})"
+          elsif item.limits.state == :BLUE
+            # If we're blue we display the blue range (GREEN_LOW - GREEN_HIGH)
+            message += " (#{values[4]} to #{values[5]})"
+          end
+        end
       else
         message = "#{packet.target_name} #{packet.packet_name} #{item.name} is disabled"
       end
-      message << " (#{packet.packet_time.sys.formatted})" if packet_time
```
Comment (jmthomas) [inline openc3/lib/openc3/microservices/decom_microservice.rb:143]: Removed the time because it's already in the table

Comment (ryanmelt) [reply openc3/lib/openc3/microservices/decom_microservice.rb:143]: I'm unsure about not including packet_time.   Received time and packet time are not necessarily the same.  Especially true for stored values (which should still alert).

**Review by ryanmelt — CHANGES_REQUESTED** [review]

**Comment by ryanmelt** [comment]
Let's discuss tomorrow.

**Review by ryanmelt — APPROVED** [review]

**Pull request merged**